elastic-agent-client.proto

```diff
@@ -9,86 +9,87 @@ package proto;
 option cc_enable_arenas = true;
 option go_package = "pkg/proto;proto";
 
-// Connection information sent to the application on startup so it knows how to connected back to the Elastic Agent.
-//
-// This is normally sent through stdin and should never be sent across a network un-encrypted.
-message ConnInfo {
-    // GRPC connection address.
-    string addr = 1;
-    // Server name to use when connecting over TLS.
-    string server_name = 2;
-    // Token that the application should send as the unique identifier when connecting over the GRPC.
-    string token = 3;
-    // CA certificate.
-    bytes ca_cert = 4;
-    // Peer certificate.
-    bytes peer_cert = 5;
-    // Peer private key.
-    bytes peer_key = 6;
+// Empty response message.
+message EmptyResponse {}
+
+// State codes for the current state.
+enum State {
+    STARTING = 0;
+    CONFIGURING = 1;
+    HEALTHY = 2;
+    DEGRADED = 3;
+    FAILED = 4;
+    STOPPING = 5;
+}
+
+// Type of unit.
+enum UnitType {
+    INPUT = 0;
+    OUTPUT = 1;
```

Comment:
Output = shipper?

Reply:
Not shipper, because this will be the GRPC to the shipper for the inputs and the actual output to elasticsearch, logstash, kafka, etc. in the shipper.